shared/main.lua

```diff
@@ -2,8 +2,6 @@ QBShared = QBShared or {}
 
 QBShared.StarterItems = {
     phone = { amount = 1, item = 'phone' },
-    id_card = { amount = 1, item = 'id_card' },
-    driver_license = { amount = 1, item = 'driver_license' },
 }
```

Comment:
Should we move this to config? It doesn't really seem appropriate as part of QBShared

Reply:
I think that's a good idea, let me go ahead and move it